doc/python_reference.rst

```diff
@@ -704,6 +704,7 @@ Source Space Data
    extract_label_time_course
    grade_to_tris
    grade_to_vertices
+   select_sources
```

Comment:
select_sources is not in the mne namespace (mne.select_sources breaks unless you add it to the mne/__init__.py)

you need to add it there for this doc link to work.

Reply:
Or leave it in `label` and put it in here as `label.select_sources` here, up to you

Reply:
+1 for label.select_sources

Reply:
It was added to the mne namespace ... but I'll remove it and let it live in `mne.label`.